uber/models/tracking.py

```diff
@@ -42,10 +42,10 @@ def track_pageview(cls):
                 return
 
             # Looking at an attendee's details
-            if "registration" in url:
+            if "registration" or "attendee" in url:
```

Comment:
This doesn't work the way you think it does.  The way you have this here, it will *always* return True.

You meant to say "if either "registration" or "attendee" is in the url, but what it actually is checking is "if 'registration' is truthy, or if 'attendee' is in the URL".

You need to say ``if 'registration' in url or if 'attendee' in url:``

Reply:
Whoops! Good catch, I'll fix both of those!